cli/src/apiToken.ts

```diff
@@ -21,13 +44,23 @@ const getAccessToken = (): Promise<string> =>
         : Promise.reject('Not logged-in, run the "login" command first.'),
   );
 
-const getOrCreateApiToken = async (
+const createApiToken = apiTokenRequest("POST", { action: "create" });
+const listApiTokens = apiTokenRequest("GET", undefined);
+
+const getOrCreateApiToken = (
```

Comment:
consider calling this from within `apiTokenRequest` which would allow you to simplify the mapping